source/includes/rest/_batch_api.md

```diff
@@ -326,3 +326,189 @@ request and the time your data appears in user interface.
     </tr>
   </tbody>
 </table>
+
+## Create or update a batch of orders
+
+> To create or update a batch of orders:
+
+```shell
+curl -X POST "https://api.getdrip.com/v2/YOUR_ACCOUNT_ID/subscribers/batches" \
```

Comment:
This should be `/orders/batches`, right?

Reply:
Yup. Fixed.